### PR TITLE
🚀 run builds on github action runner

### PR DESCRIPTION
**Pull request by n1ru4l**
Closes https://github.com/robertsLando/pkg-binaries/issues/11

### CODE REVIEW

**Comment by n1ru4l** [comment]
todo:  yaml syntax error in filter line

File: build.sh [hunk 1]
```diff
@@ -154,6 +154,8 @@ else
 	NPROC=$(nproc)
 fi
 
+NPROC=$(($NPROC+1))
```
Comment (robertsLando) [inline build.sh:157]: Could we set this to nproc*1.5 ?

Comment (n1ru4l) [reply build.sh:157]: As the people in the issue reported in the GitHub issue at some point the operations block each other e.g. for 4 cores 6 operations are slower etc. I think we are good with this formula for now (our main goal is to build on GitHub actions anyways), which does have a limit of 2 cores.

Comment (robertsLando) [reply build.sh:157]: Ok!

**Review by robertsLando — APPROVED** [review]
LGTM. Feel free to merge this when it's ready (I prefer to squash and merge to keep master cleaner)

**Pull request merged**